run/types/DeviceWrapper.ts

```diff
@@ -602,8 +595,10 @@ export class DeviceWrapper {
     if (elements && elements.length) {
       const matching = await this.findAsync(elements, async e => {
         const text = await this.getTextFromElement(e);
-        // console.info(`text ${text} lookigfor ${textToLookFor}`);
-
+        // console.info(`text ${text} lookingfor ${textToLookFor}`);
+        if (text.toLocaleLowerCase().includes(textToLookFor.toLocaleLowerCase())) {
```

Comment:
not too sure why you use `toLocaleLowerCase`, it's not as if you were dealing with multiple locales

Reply:
This is a mistake, should just be `toLowerCase()`

Although I'm not seeing that on my end...

Reply:
![Uploading Screenshot 2024-10-02 at 12.03.37 pm.png…]()

Reply:
This is a mistake, should just be `toLowerCase()`

Although I'm not seeing that on my end...

Reply:
```suggestion
        if (text.toLowerCase().includes(textToLookFor.toLowerCase())) {
```

Made suggested fix. Once committed I'll resolve this convo

Reply:
Fixed up! VSCode spazzed out for me on this one haha